client/my-sites/people/followers-list/index.jsx

```diff
@@ -192,7 +210,7 @@ const Followers = localize(
 						{ ...infiniteListConditionals }
 						key={ key }
 						items={ this.props.followers }
-						className="people-selector__infinite-list"
+						id="people-selector-infinite-list"
```

Comment:
Components really shouldn't have hard-coded ids. This is baking knowledge of how the component is used into the component and preventing us from ever having it appear twice in the DOM.

Reply:
What would be a good way to target markup in the E2E tests? (The problem with classnames is that we have strict rules about them...)

Reply:
We sometimes use a `data-e2e-something` attribute to give the tests something to hit.

Or you could give it two classes. Like `followers-list is-people`.